liquidctl/driver/asus_rog_ryujin.py

```diff
@@ -0,0 +1,81 @@
+"""liquidctl driver for the ASUS ROG RYUJIN II 360 liquid cooler.
+
+Copyright Florian Freudiger and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import logging
+
+from liquidctl.driver.usb import UsbHidDriver
+from liquidctl.util import clamp
+
+_LOGGER = logging.getLogger(__name__)
+
+_REPORT_LENGTH = 65
+_PREFIX = 0xEC
+
+_CMD_GET_STATUS = 0x99
+_CMD_GET_SPEED = 0x9A
+_CMD_SET_SPEED = 0x1A
+
+_STATUS_TEMPERATURE = "Liquid temperature"
+_STATUS_PUMP_SPEED = "Pump speed"
+_STATUS_PUMP_DUTY = "Pump duty"
+_STATUS_FAN_SPEED = "Embedded Micro Fan speed"
+_STATUS_FAN_DUTY = "Embedded Micro Fan duty"
+
+
+class RogRyujin(UsbHidDriver):
+    """ASUS ROG RYUJIN II 360 liquid cooler."""
+
+    _MATCHES = [
+        (0x0B05, 0x1988, "ASUS ROG RYUJIN II 360", {"fan_count": 1}),
+    ]
+
+    def __init__(self, device, description, **kwargs):
+        super().__init__(device, description, **kwargs)
+
+    def initialize(self, **kwargs):
+        pass
+
+    def _get_duty(self) -> (int, int):
+        """Get current pump and fan duty in %."""
+        self._write([_PREFIX, _CMD_GET_SPEED])
+        msg = self._read()
+        return msg[4], msg[5]
+
+    def get_status(self, **kwargs):
+        pump_duty, fan_duty = self._get_duty()
+        self._write([_PREFIX, _CMD_GET_STATUS])
+        msg = self._read()
+
+        return [
+            (_STATUS_TEMPERATURE, msg[3] + msg[4] / 10, "°C"),
+            (_STATUS_PUMP_SPEED, msg[6] << 8 | msg[5], "rpm"),
```

Comment:
[util.u16le_from()](https://github.com/liquidctl/liquidctl/blob/main/liquidctl/util.py#L154) could be useful here and below.